### PR TITLE
[W6.1h][W15-B3]See Jia Min Jasmine

### PR DESCRIPTION
**Pull request by JasmineSee**
Add a Printable interface and override getPrintableString() in Name, Phone, Email and Address classes

### CODE REVIEW

File: src/seedu/addressbook/data/person/Email.java [hunk 1]
```diff
@@ -6,7 +6,7 @@
  * Represents a Person's email in the address book.
  * Guarantees: immutable; is valid as declared in {@link #isValidEmail(String)}
  */
-public class Email {
+public class Email implements Printable{
```
Comment (okkhoy) [inline src/seedu/addressbook/data/person/Email.java:9]: need space between `Printable {`

File: src/seedu/addressbook/data/person/ReadOnlyPerson.java [hunk 2]
```diff
@@ -68,18 +68,26 @@ default String getAsTextHidePrivate() {
...
+            builder.append(getPrintableString(getAddress()));
         }
         builder.append(" Tags: ");
         for (Tag tag : getTags()) {
             builder.append(tag);
         }
         return builder.toString();
     }
+
+    default String getPrintableString(Printable... printables) {
```
Comment (okkhoy) [inline src/seedu/addressbook/data/person/ReadOnlyPerson.java:86]: ideally you should use this method somewhere

**Comment by okkhoy** [comment]
@JasmineSee 
Some comments added. Please ack & close the PR after reading comments.

**Comment by JasmineSee** [comment]
Received and acknowledged the comments. Thank you!

**Pull request closed without merging**